src/Card.js

```diff
@@ -33,10 +33,16 @@ export default class Card {
 	renderToHTML(md, options){
 		// unify heading levels for consistent look in anki
 		for(let i = 0; i < this.front.length; i++)
-			if(this.front[i].type == 'heading_open' || this.front[i].type == 'heading_close') this.front[i].tag = 'h1';
+			if(this.front[i].type === 'heading_open' || this.front[i].type === 'heading_close') this.front[i].tag = 'h1';
 		// render front and back to html
 		let front = md.renderer.render(this.front, md.options, {});
 		let back = md.renderer.render(this.back, md.options, {});
+
+		back = `<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/styles/default.min.css">
+				<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.1.0/highlight.min.js"></script>
+				<script>hljs.highlightAll();</script>
```

Comment:
I don't think there is any reason for the JS code being required here. The language detection and generation of the correct HTML code should already be done by the `highlight`-function of `markdown-it`. I think the only thing needed is the CSS. At least, removing the js didn't change anything in a quick test.

If you can confirm that this works as well, we should probably add the CSS during the deck initialization not depending on a CDN. If something like `import css from 'highlight.js\styles\default.css'` would work, that would be great, but unfortunately, it doesn't. Maybe you got some ideas on how to import the CSS from the package directly?

Reply:
Yeah I don't think depending on a CDN is a good idea either, I'll see what I can do!